tests/engine/storage/test_resource.py

```diff
@@ -99,7 +100,9 @@ def test_resource_caching_if_already_restored_with_different_state(
     (temporary_directory / "another_file").touch()
 
     new_storage_dir = tmp_path_factory.mktemp("new dir")
-    rasa.utils.common.copy_directory(initial_storage_dir, new_storage_dir)
+    rasa.utils.common.copy_directory(
```

Comment:
This occurence of `copy_directory` was missed here, should be replaced also with `shutil.copytree`.

Reply:
ah yes, thank you!